xml/compute.xml

```diff
@@ -759,12 +759,12 @@
    <title>memkind &mdash; heap manager for heterogeneous memory platforms and mixed memory policies</title>
    <para>
     The <emphasis>memkind</emphasis> library is a user-extensible heap manager
-    built on top of <literal>jemalloc</literal> which enables control of memory
-    characteristics and a partitioning of the heap between kinds of memory. The
-    kinds of memory are defined by operating system memory policies that have
-    been applied to virtual address ranges. Memory characteristics supported by
-    <literal>memkind</literal> without user extension include control of NUMA
-    and page size features.
+    built on top of <emphasis>jemalloc</emphasis> which enables control of
+    memory characteristics and a partitioning of the heap between kinds of
```

Comment:
Since I do not know anything in particular about what this tool is or does, I would prefer to leave this alone. I can see different interpretations of "control over" instead of "control of".